src/main/java/org/apache/maven/plugin/compiler/AbstractCompilerMojo.java

```diff
@@ -176,7 +176,7 @@
     /**
      * Set to <code>true</code> to show compilation warnings.
      */
```

Comment:
We should update such documentation to something like:
```
Set to false disable warnings during compilation
```

Reply:
I see.

Reply:
Updated.
I really missed that the opposite of flag was described by javadoc.